services/wfm/pom.xml

```diff
@@ -78,6 +77,11 @@
             <artifactId>messaging</artifactId>
             <version>${openkilda-messaging.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.openkilda</groupId>
+            <artifactId>kilda-model</artifactId>
+            <version>${openkilda-messaging.version}</version>
```

Comment:
`openkilda-messaging.version` is not appropriate here. `project.version` can be used.